cromwell_tools/cromwell_tools.py

```diff
@@ -0,0 +1,30 @@
+"""This module contains utility functions to interact with Cromwell.
+"""
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+def start_workflow(wdl_file, zip_file, inputs_file, inputs_file2, options_file, url, user, password):
+    """Use HTTP POST to start workflow in Cromwell.
+
+    :param _io.BytesIO wdl_file: wdl file.
```

Comment:
I'm wondering what these underscores are doing here.

Reply:
@rexwangcc I think you added these underscores when you documented these functions before I moved them - do they serve a purpose or should we get rid of them?

Reply:
Yes, I added the underscore here when I was documenting them. Actually, the underscore syntax(_io) here means the _io module provides the C code that the io module uses internally, according to the [source code of Python](https://github.com/python/cpython/tree/master/Modules/_io). I thought put `_io` here was more accurate, because:
```python
import io
s = io.BytesIO(b'byte')
print(s)
# <_io.BytesIO object at 0x10630d9e8>
```
The object actually is a `_io.BytesIO` instance implemented in C. 

If you think this is confused, it is better to change this back to `_io.BytesIO` for clarity.

Reply:
Okay, that makes more sense now. I'm okay with leaving the underscores then. Is that okay with you, @ambrosejcarr?